qa/mixed-cluster/src/test/java/org/opensearch/backwards/IndexingIT.java

```diff
@@ -387,7 +387,7 @@ public void testUpdateSnapshotStatus() throws Exception {
         Settings.Builder settings = Settings.builder()
             .put(IndexMetadata.INDEX_NUMBER_OF_SHARDS_SETTING.getKey(), between(5, 10))
             .put(IndexMetadata.INDEX_NUMBER_OF_REPLICAS_SETTING.getKey(), 1)
-            .putList("index.sort.field", "sortfield")
+            //.putList("index.sort.field", "sortfield") /* disabling as of now as it breaks non production scenario */
```

Comment:
@gashutos just trying  to understand if we are phasing out a valid scenario:
 - assume the index is created in 1.3 with index sort
 - the snapshot of this index is taken
 - the snapshot is restored in 2.11

That should work, right? Or the test does something different? If we comment out `index.sort.field`, the original flow that triggers the issue won't be exercised, right?